koku/api/settings/test/test_views.py

```diff
@@ -99,8 +99,9 @@ def test_post_settings_tag_enabled(self):
             ocp_list = [f"openshift-{tag}" for tag in tags]
             keys_list = aws_list + ocp_list
             max_idx = len(keys_list)
+
+        enabled_tags = list(set(random.choices(keys_list, k=random.randint(0, max_idx))))
```

Comment:
I think we may have changed the functionality of this test by removing this outside of the for loop. 

I think the initial intent was to change the enabled keys lists in each iteration to test different combinations of enabled keys. Now I think all 4 are basically running the same test. We might not need the iteration anymore, but it would be hard to tell without understanding why we moved it out.